include/SDL_float.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copying and distribution of this file, with or without modification,
+ * are permitted in any medium without royalty provided this notice is
+ * preserved.  This file is offered as-is, without any warranty.
+ * Names of contributors must not be used to endorse or promote products
+ * derived from this file without specific prior written permission.
+ */
+
+#ifndef SDL_float_h_
+#define SDL_float_h_
+
+#include <float.h>
+
+#define SDL_FLT_IS_NULL(value)                              \
+    ((value) > - FLT_EPSILON && (value) < + FLT_EPSILON)
+#define SDL_FLT_DIFF(value1, value2)            \
```

Comment:
Do we need a macro for this?

Reply:
```
#define SDL_FLT_DIFF(value1, value2)            \
    ((value1) - (value2))
#define SDL_FLT_EQUAL(value1, value2)               \
    (SDL_FLT_DIFF(value1, value2) < + FLT_EPSILON   \
     &&                                             \
     SDL_FLT_DIFF(value1, value2) > - FLT_EPSILON)
```

> Do we need a macro for this?

What does `SDL_FLT_DIFF` is used twice by the current `SDL_FLT_EQUAL`, but I agree that it is not really needed.